### ISSUE TITLE
Add automation to deploy OCP and MAS on Azure

### ISSUE THREAD

**Issue by aroute** [ibm-mas/ansible-devops#981]
Currently, there is an existing MasDevOps automation available for AWS (IPI/ROSA). A similar solution is needed for Azure (IPI/ARO). Please consider automating the solution (also need AzureDNS/LE to go with it). An idea for this request has been raised as well and the offering managers are onboard. Please note that this is not the same as what cloud vendor's Marketplace (BYOL) offers.

**Comment by nasirchouhan** [comment]
Agreed ... We need this MAS CLI tool. Please make it's priority higher ...

**Comment by durera** [comment]
This enhancement is being discussed internally for possible development this year.  We are tracking this RFE here https://ibm-ai-apps.ideas.ibm.com/ideas/MASSWAT-I-110 for now, so I am closing this issue so that we only have one place to monitor.
